src/model/model.ts

```diff
@@ -799,42 +809,45 @@ class Screen {
         }
     }
 
-    termCustomKeyHandlerInternal(e: any, termWrap: TermWrap): void {
-        if (e.code == "ArrowUp") {
+    termCustomKeyHandlerInternal(e: any, termWrap: TermWrap): void { 
+        if (GlobalModel.checkKeyPressed(e, "ArrowUp")) {
             termWrap.terminal.scrollLines(-1);
             return;
-        }
-        if (e.code == "ArrowDown") {
+        } 
+        if (GlobalModel.checkKeyPressed(e, "ArrowDown")) {
             termWrap.terminal.scrollLines(1);
             return;
-        }
-        if (e.code == "PageUp") {
+        } 
+        if (GlobalModel.checkKeyPressed(e, "PageUp")) {
             termWrap.terminal.scrollPages(-1);
             return;
-        }
-        if (e.code == "PageDown") {
+        } 
+        if (GlobalModel.checkKeyPressed(e, "PageDown")) {
             termWrap.terminal.scrollPages(1);
             return;
         }
     }
 
     isTermCapturedKey(e: any): boolean {
-        let keys = ["ArrowUp", "ArrowDown", "PageUp", "PageDown"];
-        if (keys.includes(e.code) && keyHasNoMods(e)) {
+        if(GlobalModel.checkKeyPressed(e, "ArrowUp") ||
+            GlobalModel.checkKeyPressed(e, "ArrowDown") ||
+            GlobalModel.checkKeyPressed(e, "PageUp") ||
+            GlobalModel.checkKeyPressed(e, "PageDown")) {
             return true;
         }
         return false;
     }
 
     termCustomKeyHandler(e: any, termWrap: TermWrap): boolean {
-        if (e.type == "keypress" && e.code == "KeyC" && e.shiftKey && e.ctrlKey) {
+        if (e.type == "keypress" && GlobalModel.checkKeyPressed(e, GlobalModel.KeyPressModifierShift + ":" + GlobalModel.KeyPressModifierControl + ":" + "c")) {
```

Comment:
This is wrong, need to work on the "description". description needs to be able to handle and and or